### PR TITLE
SENT fixes

### PR DESCRIPTION
**Pull request by dron0gus**
fixes https://github.com/rusefi/rusefi/issues/4685

### CODE REVIEW

**Comment by rusefillc** [comment]
24 does not match https://github.com/rusefi/rusefi/wiki/Hardware-microRusEFI-wiring

image

**Comment by dron0gus** [comment]
Fixed. Now it is 23.

File: firmware/hw_layer/drivers/sent/sent_hw_icu.cpp [hunk 1]
```diff
@@ -50,29 +50,34 @@ static ICUConfig icucfg_in1 =
 	.arr = 0xFFFFFFFFU,
 };
 
+/* current config */
+static brain_input_pin_e sentPins[SENT_INPUT_COUNT];
```
Comment (rusefillc) [inline firmware/hw_layer/drivers/sent/sent_hw_icu.cpp:54]: This falls outside of the pattern. It looks like the pattern is "activeConfiguration"

```
#if EFI_ALTERNATOR_CONTROL
	onConfigurationChangeAlternatorCallback(&activeConfiguration);
#endif /* EFI_ALTERNATOR_CONTROL */

#if EFI_BOOST_CONTROL
	onConfigurationChangeBoostCallback(&activeConfiguration);
#endif
#if EFI_ELECTRONIC_THROTTLE_BODY
	onConfigurationChangeElectronicThrottleCallback(&activeConfiguration);
#endif /* EFI_ELECTRONIC_THROTTLE_BODY */

#if EFI_ENGINE_CONTROL && EFI_PROD_CODE
	onConfigurationChangeBenchTest();
#endif

```

**Comment by rusefillc** [comment]
Damn inconsistency - sometimes not parameter but extern :(

```
void stopPedalPins() {
	brain_pin_markUnused(activeConfiguration.clutchUpPin);
	brain_pin_markUnused(activeConfiguration.clutchDownPin);
	brain_pin_markUnused(activeConfiguration.throttlePedalUpPin);
	brain_pin_markUnused(activeConfiguration.brakePedalPin);
	brain_pin_markUnused(activeConfiguration.launchActivatePin);
}
```

**Pull request merged**